### PR TITLE
Unity update

### PR DESCRIPTION
**Pull request by antonioo-c**
unity

**Issue:** #6873
**Alexa rank:** 1,962
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`   **(No Need)**
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Nothing

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
hi @antonioo-c,

thanks for the contribution.

i see some slght differences. (pink is the original one)
image

**Comment by antonioo-c** [comment]
It should be OK now, @dirien

**Comment by antonioo-c** [comment]
@dirien Can you take a look please?

**Comment by dirien** [comment]
I would ask another @simple-icons/maintainers to review it. I use the svg on this side. https://blog.unity.com/news/say-hello-to-our-new-logo-and-product-identity-system and i get still a difference.

**Review by dirien — COMMENTED** [review]
I would ask another @simple-icons/maintainers to review it. I use the svg on this side. https://blog.unity.com/news/say-hello-to-our-new-logo-and-product-identity-system and i get still a difference.

**Review by sachinraja — CHANGES_REQUESTED** [review]
I am also seeing a few slight differences compared to the one from https://brandguide.brandfolder.com/unity/downloadbrandassets. Can you update the `"guidelines"` and `"source"` in `_data/simple-icons.json` to point to `https://brand.unity.com/` and use the icon from the brandfolder (which is linked from unity's brand page)?

**Comment by antonioo-c** [comment]
OK! I'll try to work on that.

**Comment by antonioo-c** [comment]
image
After deleting the colinear-segment directly, I got this slight differences.
Could you give me some suggestions on how to eliminate this?
Thanks a lot.
@sachinraja

**Comment by antonioo-c** [comment]
I tried a few times again but found something strange...
I first downloaded the standard svg from the offical brandset you gave which I call it version1.0
secondly I adjust the size into 24x24 and deleted the text part, which i call it version2.0
finally I optimized it using SVGOMG online tool and manually deleted colinear segment, I call it version 3.0.

1.there are no difference between version1.0 and version2.0
2.there are no difference between version2.0 and version3.0
3.BUT there are that slight difference between version1.0 and version3.0, which i uploaded in the last comment.

Now I am REALLY confused, could anyone help me out?

**Review by antonioo-c — COMMENTED** [review]
forgot ,

**Comment by antonioo-c** [comment]
I don't see any differences locally and I updated the json in _data, can you take a look again? thanks a lot.
@sachinraja

**Comment by antonioo-c** [comment]
@sachinraja can you take a look please?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10361,7 +10361,8 @@
         {
             "title": "Unity",
             "hex": "000000",
```
Comment (sachinraja) [inline _data/simple-icons.json:10363]:
```suggestion
            "hex": "FFFFFF",
```
I think white is used more prominently than black.

**Review by sachinraja — CHANGES_REQUESTED** [review]
Great job, the icon is a perfect match. Just one small suggestion

**Comment by antonioo-c** [comment]
> Great job, the icon is a perfect match. Just one small suggestion

done

**Review by sachinraja — APPROVED** [review]

**Pull request merged**